dbdimp.c

```diff
@@ -3600,7 +3600,7 @@ do_bind_array_exec(sth, imp_sth, phs,utf8,parma_index,tuples_utf8_av,tuples_stat
 	OCIBindByName_log_stat(imp_sth, imp_sth->stmhp, &phs->bndhp, imp_sth->errhp,
 			(text*)phs->name, (sb4)strlen(phs->name),
 			0,
-			phs->maxlen ? (sb4)phs->maxlen : 1, /* else bind "" fails */
+			(sb4)phs->maxlen,
```

Comment:
This caught my eye.  I didn't review the context of the change, but it's worth testing a few scenarios.   E.g the one mentioned in https://github.com/php/php-src/blob/master/ext/oci8/oci8_statement.c#L1262-L1266

Reply:
This is from https://github.com/pythian/DBD-Oracle/issues/36 (unless I transcribed it in wrong, which I did in the first instance)

Im not certain if the "better" version is actually better

Reply:
I am wary of this change. Without looking through the context of the code around it, it implies a zero maxlen might be passed in which could be worse than what this change claims to fix.

Reply:
People need to look at it closely, i dont think its enough either.

Reply:
This is the original issue and diffs https://github.com/perl5-dbi/DBD-Oracle/issues/36